pkg/provision/buildroot.go

```diff
@@ -155,6 +168,14 @@ WantedBy=multi-user.target
 	}, nil
 }
 
+func rootFileSystemType(p *BuildrootProvisioner) (string, error) {
+	fs, err := p.SSHCommand("df --output=fstype / | sed 1d")
```

Comment:
Consider using `tail -n 1`, as it is likely to be more obvious to the mere mortals who are not one with the stream editor.

Reply:
Upstream suggests it is the way to go, rather than silly little things like `--without-header` etc. But I could use `tail`...

* https://lists.gnu.org/archive/html/coreutils/2012-12/msg00066.html